spark-jobs/src/main/scala/filodb/downsampler/index/DSIndexJob.scala

```diff
@@ -20,12 +21,14 @@ object DSIndexJob extends StrictLogging with Instance {
   val settings = DownsamplerSettings
   val dsJobsettings = DownsamplerSettings
 
-  private val kamonTags = Map( "rawDataset" -> dsJobsettings.rawDatasetName,
-    "owner" -> "BatchIndexUpdater")
-
   private val readSched = Scheduler.io("cass-index-read-sched")
   private val writeSched = Scheduler.io("cass-index-write-sched")
 
+  val taskCounter = Kamon.counter("num-tasks")
```

Comment:
Rename to `sparkTasksStarted` ? metric name too,

Reply:
updated variable/metric names